recipes-bsp/sgx/sgx-sdk_2.18.1.bb

```diff
@@ -14,14 +14,15 @@ SRC_URI += " \
     file://0001-sdk-configure.patch              \
     file://0002-install-sysroot.patch            \
     file://0021-fix-include-path.patch           \
+    file://tlibcxx-use-gcc-is_convertible-built-in-traits.patch \
 "
 
 ### compile ###
 
 # Build environment variables
 EXTRA_OEMAKE += "OCAMLLIB='${STAGING_LIBDIR_NATIVE}/ocaml' sdk_install_pkg"
-CFLAGS:append = " -Wno-deprecated-declarations"
-CXXFLAGS:append = " -Wno-deprecated-declarations"
+CFLAGS:append = " -Wno-deprecated-declarations -Wno-error=maybe-uninitialized"
+CXXFLAGS:append = " -Wno-deprecated-declarations -Wno-error=maybe-uninitialized"
```

Comment:
why we removing this uninitialized check flag ? wont this be important to have in the code ?

Reply:
Let me fix PR and submit backported patch to fix code itself to initialize variable. Thanks for valuable comment.

Reply:
Pushed fix. Please review it.